house/views.py

```diff
@@ -1,26 +1,48 @@
-from expenses.models import Expenses
-from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE
+from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE, LOGIN_PAGE_ROUTE
 from .models import House
 from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from house.forms import HouseIDForm
+from django.core.exceptions import ValidationError
 
 
 def home_page(request):
     return render(request, HOME_PAGE_ROUTE)
 
 
 def global_page(request):
-    houses = House.objects.all()
-    categories_amounts = Expenses.average_expenses_of_houses_by_categories(houses)
```

Comment:
Why are you removing the view of the global page?😂

Reply:
oops 
fixed 👍